learningresources/api.py

```diff
@@ -222,30 +224,69 @@ def get_resource(resource_id, user_id):
     return resource
 
 
-def create_static_asset(course_id, file_handle):
+def create_static_assets(course, xml, static_dir):
     """
-    Create a static asset for a given course.
+    Create StaticAsset instances for a course.
+    """
+    if not os.path.isdir(static_dir):
+        log.debug("Static dir %s does not exist. Skipping.", static_dir)
+        return
+    # Handle videos first.
+    videos = xml.findall(".//video")
+    for video in videos:
+        subname = get_video_sub(video)
+        if subname == "":
+            continue
+        with open(os.path.join(static_dir, subname), "rb") as raw:
+            to_load = File(raw)
+            to_load.name = "subname"
+            StaticAsset.objects.create(
```

Comment:
My understanding is that we'd eventually add all the types as part of this effort. We can add everything up front, but it seemed like there were likely to be unused detritus in courses.

However, the interface you mentioned, allowing them to "shop" for arbitrary assets, makes sense. I'll import everything.